TempMail/ContentView.swift

```diff
@@ -45,11 +45,11 @@ struct ContentView: View {
                 }.padding()
                 
                 
-                List(self.email.getInbox()) { message in
+                List(self.email.getEmails()) { message in
                     NavigationLink(destination: MessageView(message: self.email.getMessageContent(id: message.id))) {
                         InboxRow(messageInfo: message)
                     }
-                }
+                }.onAppear(perform: email.loadInbox)
```

Comment:
Why is this necessary? Wouldn't calling loadInbox() in the init() method of the Email class do the same thing?